.github/workflows/release.yml

```
@@ -0,0 +1,45 @@
+name: Publish Python 🐍 distribution 📦 to PyPI
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      - name: Build a binary wheel and a source tarball
+        run: |
+          python3 -m venv build-env
+          build-env/bin/python -m pip install --no-deps --only-binary :all: --require-hashes -r build-requirements.txt
+          build-env/bin/python -m build --no-isolation
```

*(more of a habit / personal preference)* Should the calls to Python be isolated?
```suggestion
          python3 -Im venv build-env
          build-env/bin/python -Im pip install --no-deps --only-binary :all: --require-hashes -r build-requirements.txt
          build-env/bin/python -Im build --no-isolation
```

---

Done